client/components/lists/listBody.js

```diff
@@ -140,4 +175,77 @@ BlazeComponent.extendComponent({
       keydown: this.pressKey,
     }];
   },
+
+  onCreated() {
+    dropdownMenuIsOpened = false;
+  },
+
+  onRendered() {
+    const $textarea = this.$('textarea');
+    const currentBoard = Boards.findOne(Session.get('currentBoard'));
+    $textarea.textcomplete([
+      // User mentions
+      {
+        match: /\B@(\w*)$/,
+        search(term, callback) {
+          callback($.map(currentBoard.members, (member) => {
+            const username = Users.findOne(member.userId).username;
+            return username.indexOf(term) === 0 ? username : null;
+          }));
+        },
+        template(value) {
+          return value;
+        },
+        replace(username) {
+          return `@${username} `;
+        },
+        index: 1,
+      },
+
+      // Labels
+      {
+        match: /\B#(\w*)$/,
+        search(term, callback) {
+          callback($.map(currentBoard.labels, (label) => {
+            const labelName = (!label.name || label.name === '')
+                              ? label.color
+                              : label.name;
+            return labelName.indexOf(term) === 0 ? labelName : null;
+          }));
+        },
+        template(value) {
+          // add a "colour badge" in front of the label name
+          // but first, get the colour's name from its value
+          const colorName = currentBoard.labels.find((label) => {
+            return value === label.name || value === label.color;
+          }).color;
+          return (colorName && colorName !== '')
+                 ? `<div class="minicard-label card-label-${colorName}"
+                    title="${value}"></div> ${value}`
+                 : value;
+        },
+        replace(label) {
+          return `#${label} `;
+        },
+        index: 1,
+      },
+    ]);
+
+    // customize hooks for dealing with the dropdowns
+    $textarea.on({
```

Comment:
Since 8bbc69616f91e9d09302d8a00f845b74e37d535f you can now rename `textcomplete` into `escapeableTextComplete` and remove all the duplicated code (ie the `dropdownMenuIsOpened` declaration, the `EscapeActions` registration and these `$textarea.on` hooks)